{
  "totalNumberSkippingTests": 0,
  "bugType": "only_fail",
  "totalNumberErroringTests": 0,
  "repo": "Axway/iron",
  "failingModule": "/root/workspace/Axway/iron/349269585/iron-spi-aws",
  "hostname": "spirals-vortex.lille.inria.fr",
  "totalNumberFailingTests": 1,
  "error-types": [
    "mockit.internal.UnexpectedInvocation"
  ],
  "totalNumberRunningTests": 92,
  "failing-test-cases": [
    {
      "className": "io.axway.iron.spi.aws.s3.AwsS3SnapshotStoreTest",
      "failingMethods": [
        "io.axway.iron.spi.aws.s3.AwsS3SnapshotStoreTest#shouldCreateS3Bucket"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "mockit.internal.UnexpectedInvocation",
          "failureDetail": "Unexpected invocation of:\ncom.amazonaws.services.s3.AmazonS3#listObjectsV2(String, String)\n   with arguments: \"iron-bucket-3d9d670c-a043-4d20-9738-5c5dd05a87f7\", \"iron-store-08ca95d3-e360-4ac7-9e61-61d92786c1c7/snapshot\"\n   on instance: com.amazonaws.services.s3.$Impl_AmazonS3@1e9e725a\nwhen was expecting an invocation of:\ncom.amazonaws.services.s3.AmazonS3#headBucket(com.amazonaws.services.s3.model.HeadBucketRequest)\n   with arguments: any com.amazonaws.services.s3.model.HeadBucketRequest\n   on mock instance: com.amazonaws.services.s3.$Impl_AmazonS3@1e9e725a",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    }
  ]
}